packages/components/src/Autocomplete/Menu.tsx

```diff
@@ -157,20 +162,24 @@ function useOnKeyDown(
   keyName: string,
   handler: (event: KeyboardEvent) => boolean | void,
 ) {
+  const newHandler = (event: KeyboardEvent): void => {
```

Comment:
should this be wrapped in a useCallback? I am slightly concerned about re-creating this all the time when the component that uses it re-renders.